### ISSUE TITLE
Cannot read property 'html' of undefined

### ISSUE THREAD

**Issue by HactarCE** [sindrets/atom-title-bar-replacer#22]
Steps to reproduce:

1. Install the plugin
2. See error

Restarting Atom did not help.

**Atom**: 1.23.1 ia32
**Electron**: 1.6.15
**OS**: Microsoft Windows 10 Home
**Thrown From**: [title-bar-replacer](https://github.com/sindrets/atom-title-bar-replacer) package 1.8.0

### Stack Trace

Failed to activate the title-bar-replacer package

```
At Cannot read property 'html' of undefined

TypeError: Cannot read property 'html' of undefined
    at TitleBarReplacer.activate (/packages/title-bar-replacer/lib/title-bar-replacer.js:53:93)
    at Object.activate (/packages/title-bar-replacer/lib/index.js:12:31)
    at Package.module.exports.Package.activateNow (~/AppData/Local/atom/app-1.23.1/resources/app/src/package.js:255:25)
    at ~/AppData/Local/atom/app-1.23.1/resources/app/src/package.js:227:38
    at Package.module.exports.Package.measure (~/AppData/Local/atom/app-1.23.1/resources/app/src/package.js:99:21)
    at ~/AppData/Local/atom/app-1.23.1/resources/app/src/package.js:220:32
    at Package.module.exports.Package.activate (~/AppData/Local/atom/app-1.23.1/resources/app/src/package.js:217:40)
    at PackageManager.activatePackage (~/AppData/Local/atom/app-1.23.1/resources/app/src/package-manager.js:695:42)
    at config.transactAsync (~/AppData/Local/atom/app-1.23.1/resources/app/src/package-manager.js:670:36)
    at Config.module.exports.Config.transactAsync (~/AppData/Local/atom/app-1.23.1/resources/app/src/config.js:367:24)
    at PackageManager.activatePackages (~/AppData/Local/atom/app-1.23.1/resources/app/src/package-manager.js:668:23)
    at PackageManager.activate (~/AppData/Local/atom/app-1.23.1/resources/app/src/package-manager.js:647:50)
    at ~/AppData/Local/atom/app-1.23.1/resources/app/src/atom-environment.js:865:36
```

### Commands

```
```

### Non-Core Packages

```
atom-material-syntax 1.0.8 
atom-material-ui 2.0.6 
autocomplete-python 1.10.5 
behave-theme 1.7.0 
busy-signal 1.4.3 
custom-folds 1.9.5 
file-icons 2.1.15 
flexible-ui 0.10.7 
fold-lines 0.2.2 
fold-navigator 0.0.620 
fonts 3.0.3 
gruvbox-plus-syntax 1.3.1 
haskell-ghc-mod 2.2.2 
haskell-pointfree 0.2.0 
haskell-unicode-snippets 0.1.1 
hey-pane 1.1.0 
highlight-selected 0.13.1 
ide-haskell 2.2.0 
ide-haskell-cabal 2.1.0 
ink 0.7.11 
intentions 1.1.5 
language-autohotkey 0.1.0 
language-common-lisp 0.9.5 
language-computercraft 0.2.1 
language-generic-config 1.4.0 
language-haskell 1.17.0 
language-hjson 1.0.0 
language-kerboscript 0.4.2 
language-kos 0.0.2 
language-lisp 0.2.0 
language-pegjs 0.5.0 
language-racket 0.4.0 
language-scheme 1.4.0 
linter 2.2.0 
linter-flake8 2.3.0 
linter-ui-default 1.6.10 
markdown-writer 2.7.3 
monokai 0.24.0 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
nucleus-dark-ui 0.12.3 
omnisharp-atom 0.31.2 
parinfer 1.20.0 
pristine-ui 1.1.11 
project-plus 1.0.0 
proto-repl 1.4.22 
python-autopep8 0.1.3 
python-indent 1.1.4 
python-tools 0.6.9 
rainbow-delimiters 2.1.2 
ramda-syntax 1.0.4 
run-in-terminal 1.0.1 
script 3.17.3 
simple-drag-drop-text 0.3.6 
sort-lines 0.18.0 
split-diff 1.5.2 
Sublime-Style-Column-Selection 1.7.4 
sync-settings 0.8.3 
terminal-plus 0.14.5 
title-bar-replacer 1.8.0 
```

It looks like there have been several similar issues in the past.

**Comment by sindrets** [comment]
@HactarCE Yeah, I just pushed an update and I had forgot to handle an exception. It's already fixed in v1.8.1 :)

**Comment by HactarCE** [comment]
Heh, thanks for the timely update. 😃
